tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.6
+
+LABEL maintainer "RPS <devops@rpsgroup.com>"
+
+RUN mkdir -p /glider-dac-status
+RUN mkdir /glider-dac-status/logs
```

Comment:
Per comment in last PR, this can be reduced to just one Dockerfile instruction since the intermediate directories are written: `mkdir -p /glider-dac-status/logs`